### PR TITLE
fix: OSM integration

### PR DESCRIPTION
**Pull request by lad1337**
looks like the method name has changed at some point, this is all to fix OSM integration

### CODE REVIEW

**Comment by alesti** [comment]
tested, works for me :-)

**Comment by hugovk** [comment]
It only just changed this week in OSMViz 4.0.0 which supports Python 3.7+: https://github.com/hugovk/osmviz/releases/tag/4.0.0

Don't forget to update the comment:

```diff
-    # Just a wrapper for osm.createOSMImage to translate coordinate schemes
+    # Just a wrapper for osm.create_osm_image to translate coordinate schemes
```

And `requirements.txt`:

```diff
diff --git a/requirements.txt b/requirements.txt
index 0ff23f0..82dfaf6 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,2 +1,2 @@
-osmviz
+osmviz>=4
 pillow
```

---

Alternatively, if you still need to support Python 3.6 and older:

```python
        try:
            # OSMViz 4.0+
            image, bounds = osm.create_osm_image((c1.lat, c2.lat, c1.lon, c2.lon),
                                                  zoom)
        except AttributeError:
            # OSMViz < 4.0
            image, bounds = osm.createOSMImage((c1.lat, c2.lat, c1.lon, c2.lon),
                                                zoom)
```

But 3.6 is EOL this month, so I'd suggest the former.

File: heatmap.py [hunk 1]
```diff
@@ -699,7 +699,7 @@ def _get_osm_image(bbox, zoom, osm_base):
             image_manager=PILImageManager('RGB'),
             server=osm_base)
         (c1, c2) = bbox.corners()
-        image, bounds = osm.createOSMImage((c1.lat, c2.lat, c1.lon, c2.lon),
+        image, bounds = osm.create_osm_image((c1.lat, c2.lat, c1.lon, c2.lon),
                                            zoom)
```
Comment (hugovk) [inline heatmap.py:703]: Style nit:
```suggestion
        image, bounds = osm.create_osm_image((c1.lat, c2.lat, c1.lon, c2.lon),
                                              zoom)
```

**Comment by sethoscope** [comment]
Merged. Thanks for your contribution and my apologies for the slow response.

**Pull request closed without merging**